homeassistant/components/sensor/openhardwaremonitor.py

```diff
@@ -0,0 +1,212 @@
+"""
+Support for Open Hardware Monitor Sensor Platform.
+"""
+
+from threading import Timer
+from datetime import timedelta
+import logging
+import json
+
+import urllib.request
```

Comment:
Consider using the `requests` module, it'll make reading remote JSON data a bit more straightforward.